var/spack/repos/builtin/packages/tskit/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Tskit(PythonPackage):
+    """The tskit library provides the underlying functionality used to load,
+    examine, and manipulate tree sequences"""
+
+    homepage = "https://tskit.readthedocs.io/en/latest/"
+    url      = "https://pypi.io/packages/source/t/tskit/tskit-0.3.1.tar.gz"
+
+    version('0.3.1', sha256='b9c5a9b2fb62a615e389036946345ef8a35b09f1ffee541995b16f97fedb3d36')
+
+    depends_on('python@3:',     type=('build', 'run'))
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-svgwrite',   type='run')
+    depends_on('py-jsonschema', type='run')
+    depends_on('py-h5py',       type='run')
```

Comment:
Missing dependencies on attrs 19.1.0+ and numpy 1.7+

Reply:
Also, these should all be type=build/run

Reply:
Changes pushed. You're better at finding the right requirements file than me!